static-site/src/components/Groups/edit-overview-form.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useEffect, useState } from "react";
+import * as splashStyles from "../splash/styles";
+import { CenteredForm, InputButton, TextArea } from "./styles";
+
+const OVERVIEW_TEMPLATE = `---
+title:
+byline:
+website:
+showDatasets:
+showNarratives:
+---
+
+Add description of your group here.
```

Comment:
```suggestion
<!-- Replace this line with a description of your group. -->
```

or similar, so that if the template is saved as-is, the placeholder doesn't show up.

Reply:
Squashed with copy edits and other visual polish in a53b5c7ebb75363142a04d5f745c441f8d21c7c6